docs/sources/contribute/documentation.md

```diff
@@ -0,0 +1,32 @@
++++
+title = "Documentation Guidelines"
+description = "Documentation Guidelines"
+type = "docs"
+[menu.docs]
+parent = "contribute"
+weight = 2
++++
+
+# Documentation Guidelines
+
+All Grafana documentation is written using [Markdown](https://en.wikipedia.org/wiki/Markdown), and can be found in the [docs](https://github.com/grafana/grafana/tree/master/docs) directory in the [Grafana GitHub repository](https://github.com/grafana/grafana). The [documentation website](https://docs.grafana.com.)  is generated with [hugo](https://gohugo.io) which uses [Blackfriday](https://github.com/russross/blackfriday) as its Markdown rendering engine.
+
+## Structure
```

Comment:
I think this section could move later. As a new contributor, my first question would always be, "Where do I start?" I think the How do I Contribute answers that better.